website/versioned_docs/version-v3.14.x/howto.md

```diff
@@ -138,16 +138,4 @@ kubectl get constraints
 ```
 
 ### Input Review
-
-The `input.review` object stores the [admission request](https://pkg.go.dev/k8s.io/kubernetes/pkg/apis/admission#AdmissionRequest) under evaluation. It has the following fields:
```

Comment:
Please update this as well

Reply:
This is updated same as the other one. For whatever reason the display is showing the new line below the deleted bit